### ISSUE TITLE
Incorrect usage of links

### ISSUE THREAD

**Issue by bluepnwage** [Active-Outre-Mer-Dev/dev-links#1]
Basically the links in this file is meant for the actual resources **not** the routes for the website. 
https://github.com/Active-Outre-Mer-Dev/dev-links/blob/a3bf380e4c9310f625d702f86e7c3a285edae13f/src/lib/links.json#L1-L40

Also the resources should be under the correct property cause right now all of them are under the `tech` property (design tools and fonts for example shouldnt really be there).

The overall idea on how the data would be structured is like this:
```json
{
  "tech": [
    {
      "href": "https://nextjs.org",
      "label": "Next.js"
    },
    {
      "href": "https://kit.svelte.dev/",
      "label": "SvelteKit"
    }
  ],
  "design": [
    {
      "href": "https://tailwindui.com/",
      "label": "TailwindUI"
    }
  ]
}
```
So basically under the `tech` property would be an array of tech related resources, under the `design` property would be an array of design related resources etc.

Finally, the way you're trying to link to different pages is incorrect:
https://github.com/Active-Outre-Mer-Dev/dev-links/blob/a3bf380e4c9310f625d702f86e7c3a285edae13f/src/lib/links.json#L4
In SvelteKit, if you want to link to another route you only have to write `/basics`, theres no need to try and link to the specific file in the filesystem.
